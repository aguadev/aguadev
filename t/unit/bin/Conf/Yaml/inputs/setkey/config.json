{
                   "min_per_good_tiles" : "1",
                   "target_to_yield" : {
                                          "50" : "140000",
                                          "90" : "260000",
                                          "40" : "112000",
                                          "80" : "240000",
                                          "220" : "220000",
                                          "60" : "170000",
                                          "145" : "145000",
                                          "30" : "90000",
                                          "110" : "110000"
                                        },
                   "estimated_per_align" : "80",
                   "gt_deliv_src" : "/illumina/scratch/Genotyping_Data/InstituteReports/WGS_GT_Deliverables",
                   "sge_align_opts" : "-q prod.q",
                   "hostname" : "ussd-dev-lndb01.illumina.com",
                   "build_type" : "local",
                   "casava_root" : "/illumina/development/casava/CASAVA-VariantCalling-2.12a_gVCF",
                   "launch_limit" : "5",
                   "max_read2_phiX_error_rate" : "3.0",
                   "min_read1_per_q30" : "65",
                   "password" : "zorbas",
                   "info_emails" : [
                                      "pcruz@illumina.com",
                                      "bsickler@illumina.com",
                                      "sajay@illumina.com",
                                      "JBernd@illumina.com",
                                      "mye@illumina.com",
                                      "lgomez@illumina.com",
                                      "kgietzen@illumina.com",
                                      "kviaud@illumina.com",
                                      "knobuta@illumina.com"
                                    ],
                   "clia_emails" : [
                                      "pcruz@illumina.com",
                                      "ylyan@illumina.com",
                                      "sjain@illumina.com",
                                      "jhurless@illumina.com",
                                      "sajay@illumina.com"
                                    ],
                   "archive_pending" : "/illumina/scratch/services/archive/pending",
                   "align_retries" : "2",
                   "build_qc_cancer_samples" : {
                                                  "casava_coverage_autosomal_coverage" : [
                                                                                            "ge",
                                                                                            "0.98"
                                                                                          ],
                                                  "gt_gen_concordance" : [
                                                                            "ge",
                                                                            "0.991"
                                                                          ],
                                                  "exon_depth_per_10X" : [
                                                                            "ge",
                                                                            "0.95"
                                                                          ],
                                                  "per_q30" : [
                                                                 "ge",
                                                                 "0.80"
                                                               ],
                                                  "contam_hom_fraction_bad" : [
                                                                                 "le",
                                                                                 "0.05"
                                                                               ],
                                                  "depth_per_1X" : [
                                                                      "ge",
                                                                      "0.9785"
                                                                    ],
                                                  "gb_q30" : [
                                                                "ge",
                                                                "80.0"
                                                              ],
                                                  "casava_duplicates_percent" : [
                                                                                   "le",
                                                                                   "0.05"
                                                                                 ],
                                                  "consensus_exon_coverage" : [
                                                                                 "ge",
                                                                                 "0.97"
                                                                               ],
                                                  "depth_per_10X" : [
                                                                       "ge",
                                                                       "0.9485"
                                                                     ],
                                                  "exon_depth_per_1X" : [
                                                                           "ge",
                                                                           "0.97"
                                                                         ],
                                                  "insert_median" : [
                                                                       "ge",
                                                                       "240"
                                                                     ],
                                                  "gt_gen_usage" : [
                                                                      "ge",
                                                                      "0.97"
                                                                    ],
                                                  "casava_snp_het_hom_ratio" : [
                                                                                  "le",
                                                                                  "2.1"
                                                                                ]
                                                },
                   "tools" : "/illumina/scripts/WGS/tools/bin",
                   "build_launch_limit" : "5",
                   "fasta_locations" : {
                                          "NCBI37_XX" : "/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI37_XX/HumanNCBI37_XX.fa",
                                          "NCBI37_XY" : "/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI37_XY/HumanNCBI37_XY.fa",
                                          "NCBI36_XY" : "/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI36_XY/HumanNCBI36_XY.fa",
                                          "NCBI36_XX" : "/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI36_XY/HumanNCBI36_XX.fa"
                                        },
                   "casava_cloud_concurrent_limit" : "75",
                   "username" : "pcruz",
                   "avg_yield_trimmed_gb" : "35",
                   "pl_wrapper_root" : "/illumina/scripts/WGS/PL_wrapper/PLW-1.8",
                   "include_NPF" : "n",
                   "build_version" : "NCBI37",
                   "bcl_converter_bin" : "/illumina/development/casava/CASAVA-1.8.3a3/bin/configureBclToFastq.pl",
                   "min_read2_per_q30" : "65",
                   "run_root" : [
                                   "/illumina/upload/services"
                                 ],
                   "error_emails" : [
                                       "pcruz@illumina.com",
                                       "bsickler@illumina.com",
                                       "sajay@illumina.com",
                                       "knobuta@illumina.com",
                                       "amaroo@illumina.com"
                                     ],
                   "genotype_source_dir" : "/illumina/scratch/Genotyping_Data/InstituteReports/WGS_GenotypingReports",
                   "min_q30plus_yield" : "80",
                   "avg_align_yield_per_lane_Gb" : "35",
                   "target_to_lane_count" : {
                                               "50" : "5",
                                               "90" : "8",
                                               "40" : "4",
                                               "80" : "7",
                                               "220" : "5",
                                               "60" : "5",
                                               "145" : "4",
                                               "30" : "3",
                                               "110" : "3"
                                             },
                   "align_nproc_per_lane" : "4",
                   "sge_queue" : "prod.q",
                   "scratch_root" : "/illumina/scratch/services",
                   "pipeline_root" : "/illumina/software/OLB/OLB-1.9.2/bin",
                   "cloud_emails" : [
                                       "IlluminaServicesCloud@illumina.com"
                                     ],
                   "build_qc_normal_samples" : {
                                                  "casava_coverage_autosomal_coverage" : [
                                                                                            "ge",
                                                                                            "0.98"
                                                                                          ],
                                                  "gt_gen_concordance" : [
                                                                            "ge",
                                                                            "0.9924"
                                                                          ],
                                                  "exon_depth_per_10X" : [
                                                                            "ge",
                                                                            "0.95"
                                                                          ],
                                                  "per_q30" : [
                                                                 "ge",
                                                                 "0.80"
                                                               ],
                                                  "contam_hom_fraction_bad" : [
                                                                                 "le",
                                                                                 "0.05"
                                                                               ],
                                                  "depth_per_1X" : [
                                                                      "ge",
                                                                      "0.9785"
                                                                    ],
                                                  "gb_q30" : [
                                                                "ge",
                                                                "80.0"
                                                              ],
                                                  "casava_duplicates_percent" : [
                                                                                   "le",
                                                                                   "0.05"
                                                                                 ],
                                                  "consensus_exon_coverage" : [
                                                                                 "ge",
                                                                                 "0.97"
                                                                               ],
                                                  "depth_per_10X" : [
                                                                       "ge",
                                                                       "0.9485"
                                                                     ],
                                                  "exon_depth_per_1X" : [
                                                                           "ge",
                                                                           "0.97"
                                                                         ],
                                                  "insert_median" : [
                                                                       "ge",
                                                                       "240"
                                                                     ],
                                                  "gt_gen_usage" : [
                                                                      "ge",
                                                                      "0.97"
                                                                    ],
                                                  "casava_snp_het_hom_ratio" : [
                                                                                  "le",
                                                                                  "2.1"
                                                                                ]
                                                },
                   "casava_cloud_24hours_limit" : "50",
                   "bcl_nproc_per_lane" : "4",
                   "bcl_options" : "--force --ignore-missing-control --ignore-missing-stats ",
                   "build_root" : "/illumina/build/services/Projects",
                   "error_rate_read2" : "2",
                   "lab_emails" : [
                                     "JBernd@illumina.com",
                                     "mye@illumina.com",
                                     "lgomez@illumina.com",
                                     "pcruz@illumina.com",
                                     "kgietzen@illumina.com",
                                     "kviaud@illumina.com",
                                     "tmcsherry@illumina.com"
                                   ],
                   "database" : "wgspipe",
                   "debug_emails" : [
                                       "pcruz@illumina.com"
                                     ],
                   "python_tools" : "/illumina/scripts/WGS/python_tools",
                   "max_read1_phiX_error_rate" : "2.0",
                   "dbsnp_version" : "131",
                   "error_rate_read1" : "1.5"
               }