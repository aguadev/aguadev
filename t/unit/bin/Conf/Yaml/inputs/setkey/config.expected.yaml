---
align_nproc_per_lane: 4
align_retries: 2
archive_pending: '/illumina/scratch/services/archive/pending'
avg_align_yield_per_lane_Gb: 35
avg_yield_trimmed_gb: 35
bcl_converter_bin: '/illumina/development/casava/CASAVA-1.8.3a3/bin/configureBclToFastq.pl'
bcl_nproc_per_lane: 4
bcl_options: '--force --ignore-missing-control --ignore-missing-stats '
build_launch_limit: 5
build_qc_cancer_samples:
  casava_coverage_autosomal_coverage:
    - ge
    - 0.98
  casava_duplicates_percent:
    - le
    - 0.05
  casava_snp_het_hom_ratio:
    - le
    - 2.1
  consensus_exon_coverage:
    - ge
    - 0.97
  contam_hom_fraction_bad:
    - le
    - 0.05
  depth_per_10X:
    - ge
    - 0.9485
  depth_per_1X:
    - ge
    - 0.9785
  exon_depth_per_10X:
    - ge
    - 0.95
  exon_depth_per_1X:
    - ge
    - 0.97
  gb_q30:
    - ge
    - 80.0
  gt_gen_concordance:
    - ge
    - 0.991
  gt_gen_usage:
    - ge
    - 0.97
  insert_median:
    - ge
    - 240
  per_q30:
    - ge
    - 0.80
build_qc_normal_samples:
  casava_coverage_autosomal_coverage:
    - ge
    - 0.98
  casava_duplicates_percent:
    - le
    - 0.05
  casava_snp_het_hom_ratio:
    - le
    - 2.1
  consensus_exon_coverage:
    - ge
    - 0.97
  contam_hom_fraction_bad:
    - le
    - 0.05
  depth_per_10X:
    - ge
    - 0.9485
  depth_per_1X:
    - ge
    - 0.9785
  exon_depth_per_10X:
    - ge
    - 0.95
  exon_depth_per_1X:
    - ge
    - 0.97
  gb_q30:
    - ge
    - 80.0
  gt_gen_concordance:
    - ge
    - 0.9924
  gt_gen_usage:
    - ge
    - 0.97
  insert_median:
    - ge
    - 240
  per_q30:
    - ge
    - 0.80
build_root: '/illumina/build/services/Projects'
build_type: local
build_version: NCBI37
casava_cloud_24hours_limit: 50
casava_cloud_concurrent_limit: 75
casava_root: '/illumina/development/casava/CASAVA-VariantCalling-2.12a_gVCF'
clia_emails:
  - pcruz@illumina.com
  - ylyan@illumina.com
  - sjain@illumina.com
  - jhurless@illumina.com
  - sajay@illumina.com
cloud_emails:
  - IlluminaServicesCloud@illumina.com
database: wgspipe
dbsnp_version: 131
debug_emails:
  - pcruz@illumina.com
error_emails:
  - pcruz@illumina.com
  - bsickler@illumina.com
  - sajay@illumina.com
  - knobuta@illumina.com
  - amaroo@illumina.com
error_rate_read1: 1.5
error_rate_read2: 2
estimated_per_align: 80
fasta_locations:
  NCBI36_XX: '/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI36_XY/HumanNCBI36_XX.fa'
  NCBI36_XY: '/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI36_XY/HumanNCBI36_XY.fa'
  NCBI37_XX: '/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI37_XX/HumanNCBI37_XX.fa'
  NCBI37_XY: '/illumina/scratch/services/Genomes/FASTA_UCSC/HumanNCBI37_XY/HumanNCBI37_XY.fa'
genotype_source_dir: '/illumina/scratch/Genotyping_Data/InstituteReports/WGS_GenotypingReports'
gt_deliv_src: '/illumina/scratch/Genotyping_Data/InstituteReports/WGS_GT_Deliverables'
hostname: ussd-dev-lndb01.illumina.com
include_NPF: 'n'
info_emails:
  - pcruz@illumina.com
  - bsickler@illumina.com
  - sajay@illumina.com
  - JBernd@illumina.com
  - mye@illumina.com
  - lgomez@illumina.com
  - kgietzen@illumina.com
  - kviaud@illumina.com
  - knobuta@illumina.com
lab_emails:
  - JBernd@illumina.com
  - mye@illumina.com
  - lgomez@illumina.com
  - pcruz@illumina.com
  - kgietzen@illumina.com
  - kviaud@illumina.com
  - tmcsherry@illumina.com
launch_limit: 5
max_read1_phiX_error_rate: 2.0
max_read2_phiX_error_rate: 3.0
min_per_good_tiles: 1
min_q30plus_yield: 80
min_read1_per_q30: 65
min_read2_per_q30: 65
password: zorbas
pipeline_root: '/illumina/software/OLB/OLB-1.9.2/bin'
pl_wrapper_root: '/illumina/scripts/WGS/PL_wrapper/PLW-1.8'
python_tools: '/illumina/scripts/WGS/python_tools'
run_root:
  - '/illumina/upload/services'
scratch_root: '/illumina/scratch/services'
sge_align_opts: '-q prod.q'
sge_queue: prod.q
target_to_lane_count:
  110: 3
  145: 4
  220: 5
  30: 3
  40: 4
  50: 5
  60: 5
  80: 7
  90: 8
target_to_yield:
  110: 110000
  145: 145000
  220: 220000
  30: 90000
  40: 112000
  50: 140000
  60: 170000
  80: 240000
  90: 260000
tools: '/illumina/scripts/WGS/tools/bin'
username: test
